### PR TITLE
livre_batch: Fix off-by-one errors for end frame

### PR DESCRIPTION
**Pull request by tribal-tec**

### CODE REVIEW

File: apps/livreBatch/livre_batch.py [hunk 1]
```diff
@@ -104,7 +104,7 @@ def _build_sbatch_script(self, start, end):
         values = self.dict
         values['start'] = start
         values['end'] = end
-        values['num_frames'] = end - start + 1
+        values['num_frames'] = end - start
```
Comment (hernando) [inline apps/livreBatch/livre_batch.py]: This means the the frame interval given from the command line should be understood as open on the right, shouldn't it? Perhaps you should document that.

File: apps/livreBatch/livre_batch.py [hunk 2]
```diff
@@ -204,7 +204,7 @@ def _calc_ranges(i):
         # Submit job(s) for range(s)
         idx = 1
         for sub_range in ranges:
-            idx += self._submit_jobs_for_range(idx, sub_range[0], sub_range[1])
+            idx += self._submit_jobs_for_range(idx, sub_range[0], sub_range[1] + 1)
```
Comment (hernando) [inline apps/livreBatch/livre_batch.py]: I think this is not correct for the last frame if some frames were found.

Comment (tribal-tec) [inline apps/livreBatch/livre_batch.py]: It was wrong like that without that. Tested today :)

File: apps/livreBatch/livre_batch.py (hunk 1 above)
Comment (tribal-tec) [inline apps/livreBatch/livre_batch.py]: Should have been documented like that before. I found out as well that the interval is open on the right. I'll try to document it

Comment (tribal-tec) [inline apps/livreBatch/livre_batch.py]: It's already documented
``` Frames to render [start end) ```

File: apps/livreBatch/livre_batch.py (hunk 2 above)
Comment (hernando) [inline apps/livreBatch/livre_batch.py]: Now I get it, it wasn't obvious:

    ideal_range = set(range(start_frame, end_frame))

returns a closed range set, so you have to add back the last one.

**Comment by hernando** [comment]
+1

**Pull request merged**